packages/cli/test/utils/crucible/runner/childProcessRunner.ts

```diff
@@ -5,7 +5,7 @@ import {Job, JobOptions, RunnerEnv, RunnerType} from "../interfaces.js";
 
 export class ChildProcessRunner implements RunnerEnv<RunnerType.ChildProcess> {
   type = RunnerType.ChildProcess as const;
-  private logger: Logger;
+  protected logger: Logger;
```

Comment:
@nazarhussain curious what the idea of the logger is here, technically it could be passed as options
https://github.com/ChainSafe/lodestar/blob/408033e17f08a64416a6cbb74eaa85a74113edce/packages/cli/test/utils/crucible/runner/childProcessRunner.ts#L17

which is what the `DockerRunner` does
https://github.com/ChainSafe/lodestar/blob/408033e17f08a64416a6cbb74eaa85a74113edce/packages/cli/test/utils/crucible/runner/dockerRunner.ts#L10

do we wanna keep it for ease of switching in the future or remove it? I haven't found any rational or clue in the code on why the `ChildProcessRunner` doesn't use it while the `DockerRunner` uses the `Logger`